src/main/java/io/vertx/ext/dropwizard/impl/Matcher.java

```diff
@@ -15,14 +14,26 @@
 
```

Comment:
That doesn't sound ugly to me :) Can you update the PR and we'll see what @vietj thinks. Thank you !